mattermost/mattermost-mobile
# Mm 15570 - Add Webdriver.io for E2E tests

#### Summary
Add WDIO for the e2e test framework. Replicates the original e2e test using wd/tape. Right now, it's a sub project with it's own package.json. 

##### To Test:

You need to have a simulator installed named "Pixel_8.1". We are going to address auto-installing simulators in the future.

Run the below commands (with Mattermost_Beta.apk in the main root directory of mattermost-mobile) and the simulator should start, the test will run, and then the simulator will close.

```
cd wdio
npm install
npm run test:android
```

#### Ticket Link
[JIRA](https://mattermost.atlassian.net/browse/MM-15570)
